src/extension/wps/src/test/java/au/org/emii/wps/gogoduck/parameter/TextCsvConverterTest.java

```diff
@@ -23,16 +24,14 @@
     public void testCsvGeneration() throws IOException {
         TextCsvConverter converter = new TextCsvConverter();
 
-        Path outputFile = null;
+        Path outputFile = Files.createTempFile("converter", ".csv");
+        Files.deleteIfExists(outputFile);
 
         try {
             CoordinateAxis coordinateAxis = mock(CoordinateAxis.class);
             when(coordinateAxis.getShortName()).thenReturn("TIME");
 
-            FileMetadata fileMetadata = mock(FileMetadata.class);
-            when(fileMetadata.getTime()).thenReturn(coordinateAxis);
-
-            outputFile = converter.convert(TIME_SERIES_INPUT_FILE, fileMetadata);
+            converter.convert(TIME_SERIES_INPUT_FILE, outputFile);
             assertTrue("CSV file generated differs from expected file", FileUtils.contentEquals(outputFile.toFile(), TIME_SERIES_OUTPUT_FILE));
         } finally {
             if (outputFile != null) {
```

Comment:
yes, this condition can be removed

can change to Files.deleteIfExists although we are expecting the file to be there so I'm not fussed.